From 11129d36669a3efee5dd0d49f969f11c42764f9d Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date: Mon, 29 Oct 2018 15:55:42 +0000
Subject: [PATCH] staging: bcm2835-camera: Fix alignment should match
 open parenthesis

Fix up checkpatch "Alignment should match open parenthesis" errors

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>
---
 .../staging/vc04_services/bcm2835-camera/bcm2835-camera.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/drivers/staging/vc04_services/bcm2835-camera/bcm2835-camera.c
+++ b/drivers/staging/vc04_services/bcm2835-camera/bcm2835-camera.c
@@ -1934,7 +1934,7 @@ static int bcm2835_mmal_probe(struct pla
 		ret = bm2835_mmal_init_controls(dev, &dev->ctrl_handler);
 		if (ret < 0) {
 			v4l2_err(&dev->v4l2_dev, "%s: could not init controls: %d\n",
-				__func__, ret);
+				 __func__, ret);
 			goto unreg_dev;
 		}
 		dev->v4l2_dev.ctrl_handler = &dev->ctrl_handler;
@@ -1944,7 +1944,7 @@ static int bcm2835_mmal_probe(struct pla
 		ret = mmal_init(dev);
 		if (ret < 0) {
 			v4l2_err(&dev->v4l2_dev, "%s: mmal init failed: %d\n",
-				__func__, ret);
+				 __func__, ret);
 			goto unreg_dev;
 		}
 		/* initialize queue */
@@ -1966,7 +1966,7 @@ static int bcm2835_mmal_probe(struct pla
 		ret = bm2835_mmal_init_device(dev, &dev->vdev);
 		if (ret < 0) {
 			v4l2_err(&dev->v4l2_dev, "%s: could not init device: %d\n",
-				__func__, ret);
+				 __func__, ret);
 			goto unreg_dev;
 		}
 
@@ -1976,7 +1976,7 @@ static int bcm2835_mmal_probe(struct pla
 		ret = mmal_setup_components(dev, &default_v4l2_format);
 		if (ret < 0) {
 			v4l2_err(&dev->v4l2_dev, "%s: could not setup components: %d\n",
-				__func__, ret);
+				 __func__, ret);
 			goto unreg_dev;
 		}
 
